tests/basic.test.tsx

```diff
@@ -459,7 +465,7 @@ it('can subscribe to the store', () => {
   expect(listener).toHaveBeenCalledTimes(1)
   expect(listener).toHaveBeenCalledWith(1)
   expect(listener2).toHaveBeenCalledTimes(1)
-  expect(listener2).toHaveBeenCalledWith(1)
+  expect(listener2).toHaveBeenCalledWith(1, 0)
```

Comment:
I rebased my branch and only this test was failing, but I guess that's expected so I changed it.